terraform/production/main.tf

```diff
@@ -54,7 +59,7 @@ module "api_network_route" {
 #       https://cloud.gov/docs/services/external-domain-service/#how-to-create-an-instance-of-this-service
 ###########################################################################
 module "domain" {
-  source = "github.com/18f/terraform-cloudgov//domain?ref=v0.7.1"
+  source = "github.com/18f/terraform-cloudgov//domain?ref=v0.7.1" # TODO: upgrade this
```

Comment:
Is this being flagged to be updated after this is merged?  I'm not opposed, just wanted to make sure I understand why. 🙂

Reply:
Correct. I don't understand the `domain` item well enough to feel confidant about changing it yet. But, I didn't want it to get forgotten about. So, a note for the future.

Reply:
Thank you!